common/component/kafka/kafka.go

```diff
@@ -55,6 +67,39 @@ type Kafka struct {
 	consumeRetryInterval       time.Duration
 }
 
+type SchemaType int
+
+const (
+	None SchemaType = iota
+	Avro
+)
+
+type SchemaCacheEntry struct {
+	schema         *srclient.Schema
+	codec          *goavro.Codec
+	expirationTime time.Time
+}
+
+func GetValueSchemaType(metadata map[string]string) (SchemaType, error) {
+	schemaTypeStr, ok := metadata[valueSchemaType]
```

Comment:
Please use this method from dapr/kit which also supports case-insensitive matching: https://github.com/dapr/kit/blob/77f7f031c92afb648d24d6dea18cf80e07fe21d5/metadata/utils.go#L26